tests/stl/pair_check.cpp

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Define the template classes you are using in this test on top so that it can be included in our coverage

Reply:
Like forward definition?

Reply:
yeah, for example, it will be like 

```
template class Pair<uint16_t>;
```
Try it out and then try using the coverage option from wmake and see if you get necessary coverage